config/initializers/devise.rb

```diff
@@ -17,5 +17,5 @@
 
   config.reset_password_within = 6.hours
 
-  config.sign_out_via = :delete
+  config.sign_out_via = :get
```

Comment:
If it works, it works. Keep this in mind if you run into any unexplainable bugs surrounding sign out in the future!